tests/conftest.py

```diff
@@ -1,8 +1,8 @@
 import pytest
 
-from dvc_objects.fs import MemoryFileSystem
+from dvc_objects.fs import as_filesystem
 
 
-@pytest.fixture(autouse=True)
-def memfs():
-    return MemoryFileSystem(global_store=False)
```

Comment:
I replaced this with the fsspec implementation. This fixture was only being used by in the odb tests (`test_odb.py`) so it should not really matter. We did lose the possibility of using `global_store`, but since we're using the `memory_path` fixture from `pytest-servers` to generate temporary directories in the fsspec memory file system, it should be ok